src/client/sandbox/cookie.js

```diff
@@ -32,7 +32,7 @@ export default class CookieSandbox extends SandboxBase {
         // NOTE: We must add cookie path prefix to the path because the proxied location path defferent from the
```

Comment:
We must add the cookie path prefix to the path because the proxied location path differs from the